src/components/campaignguide/ScenarioStepComponent/StoryStepComponent.tsx

```diff
@@ -21,6 +25,13 @@ export default function StoryStepComponent({ step, width }: Props) {
       <View style={
         step.border ? [space.paddingSideL, space.paddingTopM] : []
       }>
+        <View style={space.marginTopM}>
```

Comment:
So this means you'll see multiple play options during setup, depending on how many branches there are?  It only has the narrative text, so I can see needing to pause from time to time (for other intermediate instructions)?

Also should probably guard this with a `hasDissonantVoices` check.